src/tracker.js

```diff
@@ -84,94 +94,138 @@ function tracker(token, accountName, globalFunctionName, datasourceName, host) {
     }, t)
   }
 
-  delayUpload(TIMEOUT, MAX_RETRIES)
-
-  tracker.flush = uploadEvents
+  function flush() {
+    uploadEvents()
+  }
 
-  window[globalFunctionName] = function () {
+  function addEvent() {
+    var argsArray = Array.prototype.slice.call(arguments)[0]
     var ev = [
       dateFormatted(),
       session,
       accountName,
       userCookie,
       document.location.href,
       navigator.userAgent
-    ].concat(Array.prototype.slice.call(arguments))
+    ].concat(argsArray)
     if (ev.length < TRACKER_COLUMNS) {
       ev = ev.concat(Array(TRACKER_COLUMNS - ev.length).fill(''))
```

Comment:
Have we considered what are we going to do in case we've more columns? One of the options we mentioned was to change the current tracker data source and make it more 'flexible' to these variables. Instead of that, what we could have is an 'extra' column for those extra arguments a user may want to add. So we could have date, session, accountName etc as the 'default' ones, and then another column with the extra ones. What do you think? Maybe it's too soon for this change, so we could consider it for a future improvement.

Reply:
from my point of view: it's too soon and it can be fixed using more than one insert or using last field as a JSON_

I think if you have more than 8 params the problem is not the 8 params. Limits are good.